eng/scripts/automation/GenerateAndBuildLib.ps1

```diff
@@ -740,59 +740,101 @@ function GeneratePackage()
     $result = "succeeded"
 
     # Generate Code
-    Write-Host "Start to generate sdk $projectFolder"
+    # Write-Host "Start to generate sdk $projectFolder"
     $srcPath = Join-Path $projectFolder 'src'
     if (!$skipGenerate) {
+        # verify the existence of tsp-location.yaml and autorest.md
+        Write-Host "Start to generate sdk $projectFolder"
         if($specRepoRoot -eq "") {
```

Comment:
Should we also use tsp-client here for the generate command as well?

Reply:
Hello @weshaggard This script will service both data-plane and management-plane and SDK from swagger and typespec. 
we embedded `tsp-client` in `dotnet build` and for ci automation script, if the SDK is generated from typespec, we will use `tsp-client init` to preprocess and generate the SDK and set `$skipGenerate: $true` to skip generate SDK here.

So for SDK from typespec, we will use `tsp-client` to generate code.